Doc/whatsnew/3.7.rst

```diff
@@ -158,80 +290,30 @@ effort will be made to add such support.
        PEP written by Erik M. Bray; implementation by Masayuki Yamamoto.
 
 
-PEP 562: Customization of access to module attributes
+.. _whatsnew37-pep562:
+
+PEP 562: Customization of Access to Module Attributes
 -----------------------------------------------------
 
-It is sometimes convenient to customize or otherwise have control over access
-to module attributes.  A typical example is managing deprecation warnings.
-Typical workarounds are assigning :attr:`~instance.__class__` of a module
-object to a custom subclass of :class:`types.ModuleType` or replacing the
-:data:`sys.modules` item with a custom wrapper instance.  This procedure is
-now simplified by recognizing ``__getattr__`` defined directly in a module
-that would act like a normal :meth:`__getattr__` method, except that it will
-be defined on module *instances*.
+Python 3.7 allows defining :meth:`__getattr__` on modules and will call
+it whenever a module attribute is accessed.
```

Comment:
Slightly misleading; like object `__getattr__`, module `__getattr__` is called only if the attribute is otherwise not found. So it's not called "whenever a module attribute is accessed." Suggested re-word: "...and will call it whenever a module attribute is otherwise not found."